### ISSUE TITLE
Tests for IsRequestURI and IsRequestURL fail

### ISSUE THREAD

**Issue by TheTincho** [asaskevich/govalidator#118]
Hi,

The tests for govalidator are failing with go 1.6: 

```
--- FAIL: TestIsRequestURL (0.00s)
	validator_test.go:684: Expected IsRequestURL("http://foo.bar#com") to be true, got false
	validator_test.go:684: Expected IsRequestURL("http://foobar.com#baz=qux") to be true, got false


--- FAIL: TestIsRequestURI (0.00s)
	validator_test.go:733: Expected IsRequestURI("http://foo.bar#com") to be true, got false
	validator_test.go:733: Expected IsRequestURI("http://foobar.com#baz=qux") to be true, got false
```

Looking at the source code, I see that this uses `ParseRequestURI` from `net/url`, which states in the [documentation](https://golang.org/pkg/net/url/#ParseRequestURI):

> The string rawurl is assumed not to have a #fragment suffix. (Web browsers strip #fragment before sending the URL to a web server.)

So, I believe these URLs should not validate.

**Comment by annismckenzie** [comment]
I think you're right.

**Comment by asaskevich** [comment]
Fixed in #123
